config/provisioners/natss/provisioner.yaml

```diff
@@ -0,0 +1,125 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: eventing.knative.dev/v1alpha1
+kind: ClusterChannelProvisioner
+metadata:
+  name: natss
+spec: {}
+
+---
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: natss-controller
+  namespace: knative-eventing
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: natss-controller
+rules:
+  - apiGroups:
+      - eventing.knative.dev
+    resources:
+      - channels
+      - clusterchannelprovisioners
+    verbs:
+      - get
+      - list
+      - watch
+      - update
+  - apiGroups:
+      - "" # Core API group.
+    resources:
+      - services
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+  - apiGroups:
+      - networking.istio.io
+    resources:
+      - virtualservices
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: natss-channel-controller
+  namespace: knative-eventing
+subjects:
+  - kind: ServiceAccount
+    name: natss-controller
+    namespace: knative-eventing
+roleRef:
+  kind: ClusterRole
+  name: natss-controller
+  apiGroup: rbac.authorization.k8s.io
+
+---
+
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: natss-controller
+  namespace: knative-eventing
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      clusterChannelProvisioner: natss
+      role: controller
+  template:
+    metadata:
+      labels: *labels
+    spec:
+      serviceAccountName: natss-controller
+      containers:
+        - name: controller
+          image: github.com/knative/eventing/pkg/provisioners/natss/controller
+
+---
+
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: natss-dispatcher
+  namespace: knative-eventing
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      clusterChannelProvisioner: natss
+      role: dispatcher
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+      labels: *labels
+    spec:
+      serviceAccountName: natss-controller
```

Comment:
Ideally this would use a different service account (as it needs fewer permissions than the controller). Doesn't need to be this PR, but please add a TODO to use a different service account.

Reply:
The dispatcher uses now "natss-dispatcher" as ServiceAccount
Done